sdk/transaction.go

```diff
@@ -36,6 +32,106 @@ func (txs *TransactionService) GetTransaction(ctx context.Context, id string) (T
 	return MapTransaction(&b, txs.client.GenerationHash())
 }
 
+// returns an array of Transaction's for passed array of transaction ids or hashes
```

Comment:
Golang's commenting style requires always starting comments with the name of a method. Pls, change that here and in other methods as well.

Reply:
Note: this rule applies only to public methods/funcs, thus others who start with a small letter might have any, likely helpful, comment.